README.md

```diff
@@ -84,13 +84,24 @@ $ cabal build curiosity
 $ ./dist-newstyle/build/x86_64-linux/ghc-8.6.5/curiosity-0.1.0.0/x/curiosity/build/curiosity/cty serve
 ```
 
-And finally, we can build a binary with Nix:
+We can build a binary with Nix:
 
 ```
 $ nix-build -A curiosity
 $ ./result/bin/cty serve
 ```
 
+And finally, we can run a local dev environment containing curiosity and a Nginx reverse proxy using:
```

Comment:
This appears as the top comment on the PR, but read it as the last one:

Oh, you actually call it a "local dev environment" here (instead of "integration" or "tests"), that seems the right thing.